pom.xml

```diff
@@ -584,6 +585,55 @@ https://gitbox.apache.org/repos/asf?p=maven-pmd-plugin.git;a=blob_plain;f=src/ma
         <artifactId>maven-resources-plugin</artifactId>
         <version>3.3.1</version>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>license-maven-plugin</artifactId>
+        <version>2.4.0</version>
+        <configuration>
+          <acceptPomPackaging>true</acceptPomPackaging>
+          <licenseName>apache_v2</licenseName>
+          <failOnBlacklist>true</failOnBlacklist>
+          <failOnMissing>true</failOnMissing>
+          <excludedScopes>test</excludedScopes>
+          <includedLicenses>
+            <includedLicense>The MIT License</includedLicense>
+            <includedLicense>Apache Software License - Version 2.0</includedLicense>
+            <includedLicense>ISC</includedLicense>
+            <includedLicense>The BSD 3-Clause License</includedLicense>
+            <includedLicense>WTFPL</includedLicense>
+            <includedLicense>CC-BY</includedLicense>
+            <includedLicense>CC0</includedLicense>
+            <includedLicense>Unlicense</includedLicense>
+            <includedLicense>Public Domain</includedLicense>
+
+            <!-- other licenses as needed -->
+            <includedLicense>Eclipse Distribution License - v 1.0</includedLicense>
+            <includedLicense>Bouncy Castle Licence</includedLicense>
+
+            <!-- licenses for which action needed! -->
+
```

Comment:
What action is needed and where is such action represented in this PR? Or is this a leftover?

Reply:
We can't include a dependency with the license 'CDDL + GPLv2 with classpath exception'. So we need to swap out javax.annotation:javax.annotation-api for another option or maybe version with a different license.

Reply:
Just to add, the dependency in question is only used for the annotation `@javax.annotation.Generated`.

Reply:
> @javax.annotation.Generated

It's currently only in `orchestration` right?

Luckily it will be removed soon with upcoming PR from Charles 👍 #186